edx/edx-enterprise
# [ENT-979] Add the `progress_v2` option, to be used by data API fetching.

**Description:** Since we want to gradually move customers over to the new way of gathering progress data rather than all of a sudden, we introduce a versioned choice for gathering progress data -- version 2 will be used for getting it from the analytics data API, and version 1, which is the current default and to be deprecated slowly, will be getting it from Vertica.

**JIRA:** https://openedx.atlassian.net/browse/ENT-979

**Merge deadline:** ASAP

**Test Instructions**:

Nothing major -- just run the migrations to ensure they work, and that `progress_v2` is an option.

See the corresponding edx-enterprise-data PR for actual test instructions: https://github.com/edx/edx-enterprise-data/pull/34
